src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Kanban Board Capstone Project</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto&display=swap">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-lg">
+    <div class="container">
+      <a class="navbar-brand d-flex align-items-center text-uppercase text-dark">
+        <svg fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M3.055 11H5a2 2 0 012 2v1a2 2 0 002 2 2 2 0 012 2v2.945M8 3.935V5.5A2.5 2.5 0 0010.5 8h.5a2 2 0 012 2 2 2 0 104 0 2 2 0 012-2h1.064M15 20.488V18a2 2 0 012-2h3.064M21 12a9 9 0 11-18 0 9 9 0 0118 0z"></path></svg>
+        <p class="mb-0">Globe App</p>
+      </a>
+      <ul class="navbar-nav">
+        <li class="nav-item">Countries (<span id="country-counter"></span>)</li>
+        <li class="nav-item mx-md-4">Population</li>
+        <li class="nav-item">States</li>
+      </ul>
+    </div>
+  </nav>
+
+  <div class="container my-4">
+    <div class="row gx-5 gy-5 country-list"></div>
+  </div>
+
+  <footer class="p-3 m-0 border">
+    <div class="footer-container container text-dark mx-auto w-100">
+      <p class="footer-text">
+        Created by Kwasi & Daphine as Capstone Project for Module 2
+      </p>
+    </div>
+  </footer>
+
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+  <script src="./index.js"></script>
```

Comment:
- [ ] you guys have an error in your website caused by calling index.js in your index.html file, since we are using webpack, it will be responsible for bundling & calling the index.js inside the HTML file, so just remove the importing line `<script src="./index.js"></script>` and the error will be fixed 👌 

![image](https://user-images.githubusercontent.com/89282221/197328973-a9274fb5-49f7-49a2-b3a3-5ad0e0a358c6.png)

Reply:
Thank @ichala for pointing that out. We have struggled to figure out where that was coming from🙏